src/site/content/en/blog/sign-up-form-best-practices/index.md

```diff
@@ -275,7 +268,7 @@ Make sure to [salt and hash](https://cheatsheetseries.owasp.org/cheatsheets/Pass
 
 ## Don't force password updates {: #no-forced-password-updates}
 
-[Don't force users to update their passwords arbitrarily.](https://pages.nist.gov/800-63-3/sp800-63b.html#-5112-memorized-secret-verifiers:~:text=Verifiers%20SHOULD%20NOT%20require%20memorized%20secrets%20to%20be%20changed%20arbitrarily%20(e.g.%2C%20periodically).)
+[Don't force users to update their passwords arbitrarily.](<https://pages.nist.gov/800-63-3/sp800-63b.html#-5112-memorized-secret-verifiers:~:text=Verifiers%20SHOULD%20NOT%20require%20memorized%20secrets%20to%20be%20changed%20arbitrarily%20(e.g.%2C%20periodically).>)
```

Comment:
are the angle brackets correct here?

Reply:
No. Not sure where this came from — looks OK in the index.md file as it is at the moment.

Reply:
Sorry, my editor autoformats on save. I'll try to undo this.